src/components/BoemlyFormControl/BoemlyFormControl.tsx

```diff
@@ -120,6 +131,8 @@ export const BoemlyFormControl: React.FC<BoemlyFormControlProps> = ({
         return <DatePicker size={size} {...datePickerProps} />;
       case 'Textarea':
         return <Textarea bgColor="white" {...textareaProps} />;
+      case 'Slider':
+        return <Slider onChange={() => null} ariaLabel="slider" {...sliderProps} />;
```

Comment:
Hmm, I don't like those "default values". But I'm not sure yet what's the best solution for this. Will think about it and comment later ....

Reply:
```suggestion
      case 'Slider':
        return <Slider onChange={() => undefined} ariaLabel={label || 'slider'} {...sliderProps} />;
```

What about that? I can't find a better solution 🤔

Reply:
Works for me 👍